african-market-place/src/components/pages/Login/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+
+export const LoginPage = () => {
+  const initialForm = {
+    username: '',
+    password: '',
+  };
+
+  const [form, setForm] = useState(initialForm);
+
+  const onChange = e => {
+    const { name, value } = e.target;
+    setForm({
+      ...form,
+      [name]: value,
+    });
+  };
+
+  const onSubmit = e => {
+    e.preventDefault();
+    // redirect to home page on successful validation
```

Comment:
set up a blank axios call here with a promise that just console.logs response and error

Reply:
okay.  Joe is going to take over and handle the rest of the changes.